Library/Homebrew/dev-cmd/typecheck.rb

```diff
@@ -39,11 +39,15 @@ class Typecheck < AbstractCommand
         conflicts "--lsp", "--update-all"
         conflicts "--lsp", "--fix"
 
-        named_args :none
+        named_args :tap
       end
 
       sig { override.void }
       def run
+        if (args.dir.present? || args.file.present?) && args.named.present?
+          raise UsageError, "Cannot use `--dir` or `--file` when specifying a tap."
```

Comment:
```suggestion
        if (args.dir.present? || args.file.present?) && args.named.present?
          raise UsageError, "Cannot use `--dir` or `--file` when specifying a tap."
        elseif args.fix? && args.named.present?
          raise UsageError, "Cannot use `--fix` when specifying a tap."
```


Given `--fix` may try to correct Homebrew/brew code rather than the tap.